src/server/start.js

```diff
@@ -1,8 +1,5 @@
 const mongoose = require('mongoose');
 
-// Import variables.env file
-require('dotenv').config({ path: 'src/server/variables.env' });
-
```

Comment:
This was originally here so the server recognized where to find the .env file?

So if this is gone, how does the server know where to pick up the .env file and the variables in it?

Reply:
Ok, I see now. The variables are being referenced by `process.env.nameOfVariable` in the app. 

We are adding those variables to the environment through docker run --env-file  _filename_

Reply:
In the `Makefile` the `--env-file variables.env` makes that file available to Node on `process.env`

https://github.com/AntonEmery/chord-app/blob/9884d8d2b83cc143436457b8f803383c2908ddd3/Makefile#L8